components/MainHeader.tsx

```diff
@@ -93,55 +93,10 @@ const MainHeader: React.FC<{
   if (process.env.API_HOST !== 'https://api.slope.so') {
     availableCustomerTypes = [
       ...availableCustomerTypes,
-      { label: 'Qualified', value: CustomerType.PREQUALIFIED, group: '' },
-      { label: 'Compliance Onboarding', value: CustomerType.COMPLIANCE_ONBOARDING, group: '' },
-      { label: 'Ineligible', value: CustomerType.INELIGIBLE, group: '' },
-      { label: 'Prefilled Identity', value: CustomerType.MASK_KYB, group: '' },
-      {
-        label: 'Order max',
-        value: CustomerType.ORDER_MAX,
-        group: 'Order rejection reasons',
-      },
-      {
-        label: 'Order min',
-        value: CustomerType.ORDER_MIN,
-        group: 'Order rejection reasons',
-      },
-      {
-        label: 'Outstanding orders',
-        value: CustomerType.OUTSTANDING,
-        group: 'Order rejection reasons',
-      },
-      {
-        label: 'Overdue',
-        value: CustomerType.OVERDUE,
-        group: 'Order rejection reasons',
-      },
-      {
-        label: 'Same limit',
-        value: CustomerType.FORCE_REEVAL,
-        group: 'Reevaluation possibilities',
-      },
-      {
-        label: 'Increased',
-        value: CustomerType.FORCE_REEVAL_INCREASED,
-        group: 'Reevaluation possibilities',
-      },
-      {
-        label: 'Decreased, approved',
-        value: CustomerType.FORCE_REEVAL_DECREASE_APPROVED,
-        group: 'Reevaluation possibilities',
-      },
-      {
-        label: 'Decrease, rejected',
-        value: CustomerType.FORCE_REEVAL_DECREASE_REJECTED,
-        group: 'Reevaluation possibilities',
-      },
-      {
-        label: 'Ineligible',
-        value: CustomerType.FORCE_REEVAL_INELIGIBLE,
-        group: 'Reevaluation possibilities',
-      },
+      { label: 'Skip Pre Qualify', value: CustomerType.SKIP_PRE_QUALIFY, group: '' },
```

Comment:
Nit, I think the label here should still be "Qualified/Pre-qualified"

Reply:
Sure will do